packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -659,7 +655,7 @@ class ThemeData with Diagnosticable {
   /// this theme is localized using text geometry using [ThemeData.localize].
   factory ThemeData.light() => ThemeData(brightness: Brightness.light);
 
-  /// A default dark theme with a teal accent color.
+  /// A default dark theme with a teal secondary [ColorScheme] color.
```

Comment:
Maybe more explicit here?

```suggestion
  /// A default dark theme with a teal [ColorScheme.secondary] color.
```

Reply:
Done.